internal/common/db.go

```diff
@@ -23,4 +23,10 @@ type Cluster struct {
 
 	// The lease acquired for API vip
 	IngressVipLease string `gorm:"type:text"`
+
+	// Name of the KubeAPI resource
+	KubeKeyName string `gorm:"type:text"`
```

Comment:
you don't need `gorm:"type:text"`, give it a json name instead `json:"kube_key_name"`, `json:"kube_key_namespace"`